File: Modul-2/Soal 4/PRAK204-2410817120019-NadyaRamadhani.py
#input output pertama
jarijari = float(input())
tinggi = float(input())
volume = 22 / 7 * (jarijari**2) * tinggi
luas = 2 * 22 / 7 * jarijari * (jarijari + tinggi)
keliling = 2 * 22 / 7 * jarijari
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

print("")

#input output kedua
jarijari, tinggi = map(float, input().split())
volume = 22 / 7 * (jarijari**2) * tinggi
luas = 2 * 22 / 7 * jarijari * (jarijari + tinggi)
keliling = 2 * 22 / 7 * jarijari
print(f"Volume = {volume:.2f}")
print(f"Luas = {luas:.2f}")
print(f"Keliling = {keliling:.2f}")

File: Modul-2/Soal 4/PRAK204-2410817120019-NadyaRamadhani.c
#include <stdio.h>
int main(){
//input output pertama
float jarijari;
float tinggi;
float volume;
float luas;
float keliling;

printf("");
scanf("%f", &jarijari);
printf("");
scanf("%f", &tinggi);

volume = (22.0/7) * (jarijari * jarijari) * tinggi;
luas = 2 * (22.0/7) * jarijari * (jarijari + tinggi);
keliling = 2 * (22.0/7) * jarijari;

printf("Volume = %.2f\n", volume);
printf("Luas = %.2f\n", luas);
printf("Keliling = %.2f\n", keliling);

//input output kedua
printf("\n");
printf("");
scanf("%f %f", &jarijari, &tinggi);

volume = (22.0/7) * (jarijari * jarijari) * tinggi;
luas = 2 * (22.0/7) * jarijari * (jarijari + tinggi);
keliling = 2 * (22.0/7) * jarijari;

printf("Volume = %.2f\n", volume);
printf("Luas = %.2f\n", luas);
printf("Keliling = %.2f\n", keliling);

return 0;
}